integration/build_test.go

```diff
@@ -85,10 +85,14 @@ func TestBuild(t *testing.T) {
 				defer teardown()
 			}
 
-			// remove image in case it is already present
 			removeImage(t, test.expectImage)
 			skaffold.Build(test.args...).InDir(test.dir).RunOrFail(t)
 			checkImageExists(t, test.expectImage)
+
+			removeImage(t, test.expectImage)
+			skaffold.Build(append(test.args, "--cache-artifacts=true")...).InDir(test.dir).RunOrFail(t)
+			skaffold.Build(append(test.args, "--cache-artifacts=true")...).InDir(test.dir).RunOrFail(t)
```

Comment:
wait, what is the second build doing? We are not testing anything here. we can check for some log statements like 
"re-using cache artifacts.. or something"

Reply:
Yes, I'll try to improve that. I was at least checking that it wasn't failing

Reply:
I've added assertions on the output